main.scss

```
@@ -7,14 +7,196 @@
 @import "src/scss/colors";
 @import "src/scss/variables";
 @import "src/scss/mixins";
-@import "src/scss/themes";
-@import "src/scss/teaser-collection";
 
-@if ($o-teaser-collection-is-silent == false) {
- 	@include oTeaserCollection;
+/// Output all o-teaser-collection styles. Accepts an options map to include features granularly.
+/// See the README for more details of all available options.
```

Is it possible to make README a link? 🤔

---

Should we actually be linking to the sassdoc?

---

```suggestion
/// See the [README](https://registry.origami.ft.com/components/o-teaser-collection/readme) for more details of all available options.
```

Yeah we can, markdown is rendered. 👍 
The Sassdoc for these maps is rubbish, so not right now. It's something I'd like to improve post migration